cl/corpus_importer/tasks.py

```diff
@@ -1215,7 +1225,167 @@ def make_docket_by_iquery(
         d,
         tag_names,
         add_to_solr=True,
+        from_iquery_scrape=from_iquery_scrape,
+    )
+
+
+@retry((requests.Timeout, PacerLoginException), tries=3, delay=0.25, backoff=1)
+def query_iquery_page(
+    court_id: str, pacer_case_id: int
+) -> bool | dict[str, Any]:
+    """Query the iquery page for a given PACER case ID.
+
+    :param court_id: A CL court ID where we'll look things up.
+    :param pacer_case_id: The Pacer Case ID to lookup.
+    :return: False if not valid report, otherwise the report.data.
+    """
+
+    cookies = get_or_cache_pacer_cookies(
+        "pacer_scraper",
+        settings.PACER_USERNAME,
+        password=settings.PACER_PASSWORD,
+    )
+    s = ProxyPacerSession(
+        cookies=cookies,
+        username=settings.PACER_USERNAME,
+        password=settings.PACER_PASSWORD,
     )
+    report = CaseQuery(map_cl_to_pacer_id(court_id), s)
+    report.query(pacer_case_id)
+    if not report.data:
+        logger.info(
+            "No valid data found in iquery page for %s.%s",
+            court_id,
+            pacer_case_id,
+        )
+        return False
+    return report.data
+
+
+@app.task(
+    bind=True,
+    autoretry_for=(RedisConnectionError,),
+    max_retries=3,
+    interval_start=1 * 60,
+    interval_step=5 * 60,
```

Comment:
Hm, won't another be scheduled in these five minutes, and might that be an issue? Maybe it's better just to let it die?

Reply:
Yeah, that's right! I just removed the retries from the `probe_iquery_pages` task.